css/ace-mantis.css

```diff
@@ -294,6 +294,35 @@ i.datetimepicker {
     color: #333333;
 }
 
+.dropdown-menu .divider {
+    margin: 2px 0;
+}
+
+.list > li {
```

Comment:
The style changes are generic and needed for any future use of this list.js library with Mantis Modern UI. The projects-list entry is very specific to the implemented search box in a dropdown thus I qualified it. There is no right or wrong here, it is just a judgement call.